Src/AutoFakeItEasy.FakeItEasy4UnitTest/AutoFakeItEasy.FakeItEasy4UnitTest.csproj

```diff
@@ -17,7 +17,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="FakeItEasy" Version="[4.0.0]" />
+    <PackageReference Include="FakeItEasy" Version="[4.9.2]" />
```

Comment:
Not sure how you feel about this change. Me, I'd prefer all the tests to run against the latest release within the major version range, but I think this is only my opinion. In this case, the .NET Standard 2.0 package for FakeItEasy is not available until 4.9.0.

Reply:
Hi Blair,

Glad to see you 😊

I don't have strong opinion on that. Lower possible version is good because usually features are added up during a version evolution, so it could happen that we test functionality/behavior which is not available (or is different) for the lower versions.

On other hand, clients usually install the latest major version, which is especially true for the ancient versions.

To make it simple, let's prefer the consistency across the solution and not having exceptions in the particular projects. So may I ask you to please add a condition here and target `4.9.2` for `netstandard2.0` only?

If in future I end up with a different strategy, I'll update all the project at once, including this one.

Thank you a lot for your contribution! 🥂